pages/books/books.module.css

```diff
@@ -0,0 +1,16 @@
+#bookCatalog {
```

Comment:
Make sure you add `// TODO` comments any time there's an empty function/selector! Once we add linting to the GH pipelines I think this may not pass if empty

Reply:
Could you add a CSS clean up ticket? We got CSS-in-JS, CSS modules, AND material UI! Which is actually fine, but we will definitely have overlapping styles that can be simplified. (We'll deal with that later though)

Typically we want to avoid custom styles as much as possible and try to stick with Material UI's layout components and override the styles/take advantage of theming. [Box](https://mui.com/material-ui/react-box/) is a really great component for this! I think these ones can be rewritten as components with overrides; `bookCatalog-topbar` would look something like:

`<Box display="flex" flexDirection="row" flexWrap="nowrap" justifyContent="space-between">`

Reply:
Yeah, the only thing I'd like to be explicitly defined in a css file is things like  button colours so we keep the DRY, I'll get that ticket made so we can go find better solutions to those I provided here. I haven't worked with material before so I'm a bit behind on best practice, but I'll brush up a bit more on that!